packages/iracing-plugin-stream-deck/rollup.config.mjs

```
@@ -14,6 +15,66 @@ const iconsPackagePath = path.resolve(__dirname, "../icons");
+ * `piTemplatePlugin` (EJS `platform` variable), and the emitted `config.json`.
+ * Unknown keys in the local file are warned about and ignored (never merged).
+ */
+const platformFeaturesPath = path.resolve(__dirname, "platform-features.json");
+const localFeaturesPath = path.resolve(__dirname, "../../feature-flags.local.json");
+const committedFeatures = JSON.parse(readFileSync(platformFeaturesPath, "utf-8"));
+let platformFeatures = committedFeatures;
+if (existsSync(localFeaturesPath)) {
+	const localFeatures = JSON.parse(readFileSync(localFeaturesPath, "utf-8"));
+	const unknown = findUnknownKeys(committedFeatures, localFeatures);
+	if (unknown.length > 0) {
+		console.warn(
```

ESLint

[error] 'console' is not defined.

(no-undef)